src/System.Text.ValueBuilder/System/Variant2.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    /// <remarks>
+    /// This is a pattern we can use to create stack based spans of <see cref="Variant"/>.
+    /// </remarks>
+    public readonly struct Variant2
```

Comment:
Is htis substantively different from `ValueTuple<Variant, Variant>`?

Reply:
`ValueTuple` isn't `readonly`?